### ISSUE TITLE
Useless error message for empty overrides.yml

### ISSUE THREAD

**Issue by AlexanderFabisch** [rock-core/autoproj#137]
I have an empty overrides.yml in my project:

```yaml
# This section of the file allows to modify the VCS information that is defined
# in the package sets. Below is an example to change the branch name for a given
# package
#
overrides:
```

This results in the following error:

```
$ aup
operating system: ubuntu,debian - 16.04,16.04.2,lts,xenial,xerus
updating autoproj                                               
bundler: connected to https://rubygems.org/
updated autoproj main configuration         

no implicit conversion of nil into Array
```

The fix would be to comment the last line of the overrides.yml but the error message that I get from autoproj is not very informative.

**Comment by doudou** [comment]
Already fixed on master ... thanks for reporting anyways ;-)

**Comment by doudou** [comment]
Fixed by [2.1.0](https://github.com/rock-core/autoproj/releases/tag/v2.1.0)
